src_c/surface.c

```diff
@@ -1092,7 +1092,7 @@ surf_get_palette_at(PyObject *self, PyObject *args)
 }
 
 static PyObject *
-surf_set_palette(PyObject *self, PyObject *args)
+surf_set_palette(PyObject *self, PyObject *list)
```

Comment:
Maybe rename to sequence, just because that what it actually needs to be?

Reply:
Yup! I only chose `list` because the old code used that name. But I'll update this too in this PR